engine/source/private/core/engine.cpp

```diff
@@ -1,31 +1,47 @@
 ﻿#include "core/engine.hpp"
 #include <chrono>
 #include <iostream>
+#include <SDL2/SDL.h>
 
 Engine gEngine;
 
 void Engine::Initialize()
 {
-    std::cout << "Engine::Initialize\n";
+    // Initialize SDL and create a window
```

Comment:
Eventually this should be moved to a different function, I feel like.

Reply:
I agree, an abstraction would be best, especially with regard for the possibility for cross-platform support and support for different graphics APIs. but for now we have a simple window.